align_system/configs/adm_component/icl/phase2_comparative.yaml

```diff
@@ -16,10 +16,10 @@ icl_generator_partial:
     sort_actions: true
     most_similar_first: false
     datasets:
-      medical: /data/shared/samba/phase2_icl/June2025-MU-train_20250602.json
-      affiliation: /data/shared/samba/phase2_icl/June2025-AF-train_20250523.json
-      merit: /data/shared/samba/phase2_icl/June2025-MF-train_20250523.json
-      personal_safety: /data/shared/samba/phase2_icl/June2025-PS-train_20250602.json
-      search: /data/shared/samba/phase2_icl/June2025-SS-train_20250527.json
+      medical: align_system/resources/icl/phase2/July2025-MU-train_20250804.json
+      affiliation: align_system/resources/icl/phase2/July2025-AF-train_20250804.json
+      merit: align_system/resources/icl/phase2/July2025-MF-train_20250804.json
+      personal_safety: align_system/resources/icl/phase2/July2025-PS-train_20250804.json
+      search: align_system/resources/icl/phase2/July2025-SS-train_20250804.json
```

Comment:
These are udpated to the latest dated training json.  Not sure if that is a problem.

Reply:
I added the June20205 JSON files `phase2_comparative.yaml` was pointing too.